lib/ansible/plugins/connection/winrm.py

```diff
@@ -359,7 +359,7 @@ def put_file(self, in_path, out_path):
         if result.status_code != 0:
             raise AnsibleError(to_native(result.std_err))
 
-        put_output = json.loads(result.std_out)
+        put_output = json.loads(to_text(result.std_out))
```

Comment:
If something on the managed machine messes up, this could be in some other encoding.  Also, there's always the chance that the way the client machine handles non-utf-8 bytes will lead to those bytes coming through unescaped.  Basically, the connection plugin is a boundary between Ansible and another system.  So we need to be a little more paranoid in sanitizing our data in this area.